cmux-tui/bindings/reconcile_registry_artifact.py

```
@@ -410,20 +412,38 @@ def _npm_status(package: str, version: str, artifact: Path) -> str:
+                if (
+                    not isinstance(bootstrap_release, dict)
+                    or dist_tags.get("bootstrap") != NPM_BOOTSTRAP_VERSION
+                ):
+                    raise RegistryError(
+                        "npm bootstrap metadata is incomplete for cmux-sdk"
+                    )
+                # The one-time SDK bootstrap intentionally reserves the npm
+                # name by putting its prerelease on `latest`. The stable
+                # release path below still requires `latest == version` once
+                # the immutable stable archive exists.
+                pass
```

Ruff (0.16.1)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)